src/logics/ArrayHelpers.cc

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2023, Martin Blicha <martin.blicha@gmail.com>
+ *
+ *  SPDX-License-Identifier: MIT
+ *
+ */
+
+#include "ArrayHelpers.h"
+
+#include "TreeOps.h"
+
+vec<PTRef> collectStores(Logic const & logic, PTRef fla) {
+    class CollectStoresConfig : public DefaultVisitorConfig {
```

Comment:
As I understand we use it in LATheory and in Array theory
Why you've decided to add the processing of ALIA, ALRA, ... to the UFLA, and not to the array btw?

Reply:
The Array theory was created as the first implementation to handle QF_AX logic. However, since then I came to the conclusion that we cannot create separate theory for each logic, because of combinatorial explosion.
Instead we should try to unify what we have. My plan is to remove separate Array theory and instead put the functionality in `UFTheory`.

Reply:
Gotcha